package.json

```diff
@@ -6,7 +6,7 @@
     "lint": "standard"
   },
   "dependencies": {
-    "govuk-frontend": "^4.4.1"
+    "govuk-frontend": "4.7.0"
```

Comment:
Is it worth specifying a range? I'm not sure if that makes any difference in practice...

Reply:
thanks Richard, it's highly unlikely that 4.7.x will be release, but for peace of mind I can do ~